scripts/skills/racial/rf_zombie_racial.nut

```diff
@@ -47,6 +47,14 @@ this.rf_zombie_racial <- ::inherit("scripts/skills/skill", {
 			});
 		}
 
+		ret.push({
+			id = 12,
+			type = "text",
+			icon = "ui/icons/obituary.png",	// skull icon
+			text = "Resurrect Humans that you kill as a zombie, if possible",	// This is implemented in onActorKilled of zombie.nut
```

Comment:
I'd reword this as: `Humans killed by you will rise as Wiedergangers after some time`. The `if possible` part is unnecessary because headless wiedergangers also don't resurrect and struck down characters are not killed.